Makefile

```diff
@@ -251,9 +251,11 @@ check-licenses:
 	GOBIN=$(GOBIN) go install github.com/google/go-licenses
 	$(GOBIN)/go-licenses check --ignore golang.org/x,github.com/runfinch/finch --ignore  github.com/multiformats/go-base36 --allowed_licenses Apache-2.0,BSD-2-Clause,BSD-3-Clause,ISC,MIT --include_tests ./...
 
+COVERAGE_THRESH = 45
 .PHONY: test-unit
 test-unit:
-	go test $(shell go list ./... | grep -v e2e | grep -v benchmark | grep -v mocks) -shuffle on
+	go test -cover -coverprofile=coverage.out $(shell go list ./... | grep -v e2e | grep -v benchmark | grep -v mocks) -shuffle on
```

Comment:
We should probably exclude more files/packages.

I'm not sure if we get much value out of "testing" `pkg/version`, `pkg/system`, `pkg/fmemory`, or `pkg/flog`, or even the new `coverage` file itself.

Reply:
omitted the files.